motion_computation/src/motion_computation_worker.cpp

```diff
@@ -93,7 +93,7 @@ cav_msgs::ExternalObjectList MotionComputationWorker::predictionLogic(cav_msgs::
     switch(obj.object_type)
```

Comment:
You still have this commented out switch. Please use this or the if statements. Don't leave commented code hanging around.